tfhe/src/integer/gpu/mod.rs

```diff
@@ -32,6 +33,13 @@ pub enum PBSType {
     Classical = 1,
 }
 
+#[allow(dead_code)]
+#[repr(u32)]
+pub enum CompressionType {
+    Compress = 0,
+    Decompress = 1,
+}
```

Comment:
If you have two different scratch functions for compression & decompression you won't need this enum anymore.